.github/workflows/release-canary.yml

```
@@ -20,7 +44,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       # Ensure that the README is published with the package
       - run: rm -f packages/cli/README.md && cp README.md packages/cli
-      - run: echo "PR_VERSION=0.0.0-pr.${{github.event.pull_request.number}}.$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+      - run: echo "PR_VERSION=${{ github.event_name == 'workflow_dispatch' && inputs.release_name || '0.0.0-pr.${{github.event.pull_request.number}}'.$(git rev-parse --short HEAD)}}" >> $GITHUB_ENV
```

actionlint (1.7.4)

got unexpected character '$' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)